addons/im_livechat/static/src/core/public_web/livechat_rating_notification_service.js

```diff
@@ -0,0 +1,30 @@
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+
+import { markup } from "@odoo/owl";
+
+export const livechatRatingNotificationService = {
+    dependencies: ["bus_service", "notification"],
+
+    start(env, { bus_service, notification }) {
+        bus_service.subscribe(
+            "livechat_rating_notification",
+            ({ rating_url, visitor_name, feedback, channel_id }) => {
+                const title = visitor_name
+                    ? _t("%(name)s left a rating:", { name: visitor_name })
+                    : _t("Visitor left a rating:");
+                const chatUrl = `/odoo/discuss?active_id=discuss.channel_${channel_id}`;
+                const message = markup`
+                ${title} <img class="o_livechat_emoji_rating" src="${rating_url}"/>
+                ${markup(feedback)}<br/>
```

Comment:
It would be best as store data so the markup is automatically handled as necessary.